### DISCUSSION TITLE
Animated webp support..

### DISCUSSION THREAD

**Discussion by artnose** [ImageMagick/ImageMagick#2157] [General]
I can't seem to open this image w/ IM..    

https://res.cloudinary.com/demo/image/upload/fl_awebp/cell_animation.webp

Is there plans for animated webp support, or am I doing something wrong?

**Comment by fmw42** [comment]
Works fine for me on IM 7.0.10.19 Q16 Mac OSX with webp v1.0.3 delegate installed

`magick cell_animation.webp x.gif`

have you installed webp delegate?  What is your IM version and platform and webp delegate version?

**Comment by urban-warrior** [comment]
Animated Webp support requires the libwebpmux delegate library.  If only the libwebp delegate is found @ ImageMagick build time, only single frame Webp is supported.

**Comment by artnose** [comment]
On Centos 7..   Have this installed. 
libwebp-0.3.0-7.el7.x86_64
ImageMagick-7.0.10-10.x86_64

it includes 
./lib64/libwebpmux.so.3.0.4
./lib64/libwebpdemux.so.2
./lib64/libwebpdemux.so.2.0.6
./lib64/libwebpmux.so.3
./lib64/libwebpmux.so.0
./lib64/libwebpmux.so.0.0.0

**Comment by urban-warrior** [comment]
ImageMagick requires libpdemux, version 0.5.0 or above.

**Comment by artnose** [comment]
edit:  updated to  
ImageMagick-7.0.10-19.x86_64
libwebp-0.3.0-7.el7.x86_64
.. still getting this error.. 

convert: data encoding scheme is not supported `cell_animation.webp' @ error/webp.c/ReadWEBPImage/577.

**Comment by fmw42** [comment]
I simply installed webp from MacPorts and it works fine.

**Comment by fmw42** [comment]
As urban-warrior said - ImageMagick requires libpdemux, **version 0.5.0** or above.

**Comment by Ferdari** [comment]
How to install the library?(libwebpdemux), im newbie sorry

**Reply by fisiculturista** [reply]
Same issue here. Cannot upload an animated WebP to my server:

Error code: data encoding scheme is not supported `/tmp/imagickZsDgmE' @ error/webp.c/ReadWEBPImage/338

So I have found this topic and could not find anywhere how to install libpdemux on a linux server.

I have installed WebP using the command:

sudo apt install webp

It was installed, but the error persists.

**Reply by ghost** [reply]
> Same issue here. Cannot upload an animated WebP to my server:
> 
> Error code: data encoding scheme is not supported `/tmp/imagickZsDgmE' @ error/webp.c/ReadWEBPImage/338
> 
> So I have found this topic and could not find anywhere how to install libpdemux on a linux server.
> 
> I have installed WebP using the command:
> 
> sudo apt install webp
> 
> It was installed, but the error persists.

I had to compile ImageMagick from source to enable WebP support. Just make sure to also install `libwebp-dev` otherwise the `.\configure` won't see the libraries
